plugins/nf-schema/src/main/nextflow/validation/SchemaValidator.groovy

```diff
@@ -356,21 +379,22 @@ class SchemaValidator extends PluginExtensionPoint {
     // Beautify parameters for --help
     //
     @Function
-    String paramsHelp(
+    public String paramsHelp(
         Map options = null,
         String command
     ) {
+        // TODO add link to help message migration guide once created
+        log.warn("Using `paramsHelp()` is deprecated. Check out the help message migration guide: <url>")
```

Comment:
I don't think many pipeline users will see this since we're enforcing plugin version pinning at this point. I updated the function since your review to make it useable again so maybe I can remove this message? (although I would prefer everyone to use the new system while only keeping this function here for backwards compatibility and for very specific use cases)

Reply:
Maybe add something like "contact a maintainer of the pipeline" to the message? But this is not a blocker, I agree few people should see this message as we enforce pinning the versions

Reply:
Good idea thanks!